src/app/components/admin/featured/featured.component.ts

```diff
@@ -52,15 +52,15 @@ export class FeaturedComponent implements OnInit {
     }
   }
 
-  async getListItems() {
-    const result = await this.featuredService.getListItems(this.parent);
+  async listItemsRef() {
+    const result = await this.featuredService.listItemsRef(this.parent);
     if (result) {
       this.itemList = Object.keys(result);
     }
   }
 
-  getFeaturedItems() {
-    this.featuredService.getFeaturedItems(this.parent, this.featuredType).on('value', (snapshot) =>{
+  featuredItemsRef() {
+    this.featuredService.featuredItemsRef(this.parent, this.featuredType).on('value', (snapshot) =>{
```

Comment:
While this isn't technically using an observable, the end result is the same. We're effectively watching the data at this reference for changes, so a better name would be watchFeaturedItems()